src/main.js

```diff
@@ -15,6 +15,7 @@ const touchRegExp = /touch/;
 // 300ms is the usual mouse interval;
 // // However, an underpowered mobile device under a heavy load may queue mouse events for a longer period.
 const IGNORE_MOUSE_TIMEOUT = 2000;
+const ESC = 27;
```

Comment:
We better extract this to an enum-like entity like `KeyCode`:

https://github.com/telerik/blazor/blob/4f624d14e1ed064dc0ce769b154f258667beb769/js/telerik-blazor/src/common/navigation/keys.ts#L6

Other keys can be added to this enum and it can be reused.